Twig/Extension/FormatterMediaExtension.php

```diff
@@ -109,4 +109,14 @@ public function path($media, $format)
     {
         return $this->getTwigExtension()->path($media, $format);
     }
+
+    /**
+     * NEXT_MAJOR: Remove this method when bumping requirements to twig 2.
+     *
+     * {@inheritdoc}
+     */
+    public function getName()
```

Comment:
We could add it if we release this before SonataFormatter, but I am not sure...

IMO I see 2 ways:

- Add this method, wait for SonataFormatter and release again without this method and add to conflicts earlier versions of SonataFormatter.

- Wait for SonataFormatter release and add a conflict on earlier versions.

WDYT?